test/Board.spec.js

```diff
@@ -0,0 +1,259 @@
+/* For everyone's sanity...
+3x3
+0  1  2
+3  4  5
+6  7  8
+
+4x4
+0  1  2  3
+4  5  6  7
+8  9  10 11
+12 13 14 15
+*/
+
+QUnit.test("Board.Generate2D creates a correct adjacency matrix", function (assert) {
+	const expected = [[-1, -1, -1, -1, 0, 1, -1, 3, 4],
```

Comment:
This format is a little goofy. Consider tabulating to make it easier to see what's going on.